src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const personStatus = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
```

Comment:
Don't forget about 2 spaces
```suggestion
        <p className="Person__age">
          {`I am ${age}`}
        </p>
```